modules/dataverse-spi/pom.xml

```diff
@@ -67,19 +67,14 @@
             <id>central</id>
             <url>https://central.sonatype.com/repository/maven-snapshots/</url>
         </snapshotRepository>
```

Comment:
At https://github.com/IQSS/dataverse/actions/runs/18604319308/job/53050122472?pr=11905 I see "deploy snapshot" is failing with this:

`Error:  Failed to execute goal org.sonatype.central:central-publishing-maven-plugin:0.9.0:publish (injected-central-publishing) on project dataverse-spi: Failed to deploy artifacts: Could not transfer artifact io.gdcc:dataverse-spi:jar:2.1.0-PR11905-20251017.204025-1 from/to central (https://central.sonatype.com/repository/maven-snapshots/): status code: 401, reason phrase: Unauthorized (401) -> [Help 1]`

Hmm. 🤔